### PR TITLE
Add word break prop to typography

### PR DESCRIPTION
**Pull request by ksmort**
## Description
Add wordBreak prop to Typography.
Update tests and documents.

## Screenshots
Screenshot 2021-12-02 at 16-34-03 Typography Riipen UI

## Where to Start
components/Typography

### CODE REVIEW

File: packages/riipen-ui-docs/src/pages/components/typography/typography.md [hunk 1]
```diff
@@ -7,15 +7,18 @@ set of type sizes that work well together along with the layout grid.
 
 ## General
 
-The *Roboto* font will **not** be automatically loaded by Riipen-UI.
+The _Roboto_ font will **not** be automatically loaded by Riipen-UI.
```
Comment (ksmort) [inline packages/riipen-ui-docs/src/pages/components/typography/typography.md:10]: Not sure why this changed, I think an auto-save thing?

File: packages/riipen-ui-docs/src/pages/components/typography/typography.md [hunk 2]
```diff
@@ -36,15 +39,21 @@ You can also use the CSS native "inherit" for a wider range of colors.
...
 
 ```jsx
-{/* There is already an h1 in the page, let's not duplicate it. */}
+{
+  /* There is already an h1 in the page, let's not duplicate it. */
+}
 <Typography variant="h1" component="h2">
   h1. Heading
-</Typography>
+</Typography>;
```
Comment (ksmort) [inline packages/riipen-ui-docs/src/pages/components/typography/typography.md:56]: Ditto here, I think just autosave

File: packages/riipen-ui-docs/src/pages/components/typography/typography.md [hunk 3]
```diff
@@ -36,15 +39,21 @@ You can also use the CSS native "inherit" for a wider range of colors.
 
 {{"demo": "pages/components/typography/TextTransform.js"}}
 
+## Word Break
+
+{{"demo": "pages/components/typography/WordBreak.js"}}
```
Comment (ksmort) [inline packages/riipen-ui-docs/src/pages/components/typography/typography.md:44]: new thing

File: packages/riipen-ui/src/components/Typography.jsx [hunk 4]
```diff
@@ -127,7 +132,8 @@ class Typography extends React.Component {
     mobileBreakpoint: "sm",
     textAlign: "inherit",
     textTransform: "inherit",
-    variant: "body1"
+    variant: "body1",
+    wordBreak: "normal"
```
Comment (ksmort) [inline packages/riipen-ui/src/components/Typography.jsx:136]: Was undecided between this and breakword, or just no class added to Typography if wordBreak is not passed (i.e., no default prop)?

File: packages/riipen-ui-docs/src/pages/components/typography/typography.md (hunk 1 above)
Comment (stredger) [reply packages/riipen-ui-docs/src/pages/components/typography/typography.md:10]: yea your eslint/prettier is probs formatting stuff on save, all good

**Review by stredger — APPROVED** [review]

**Pull request merged**